src/EvtGenNtuplizer_EvtInfo.cc

```diff
@@ -180,15 +98,5 @@ EvtGenNtuplizer::FillEvent( const edm::Event& iEvent, const edm::EventSetup& iSe
   //EvtInfo.Flag_badMuon                   = *_metbadmuhandle;
   //EvtInfo.Flag_badChargedhadron          = *_metbadchadhandle;
 
-  EvtInfo.McSigTag = (
```

Comment:
This variable contents should be moved to the Flag_METFilter variables, not deleted. Please also keep the commended out Flag_bad* variables, as they are expected to be re-enabled.